server/server.go

```diff
@@ -2254,6 +2253,17 @@ func (s *Server) ReadyForConnections(dur time.Duration) bool {
 	return false
 }
 
+// Quick utility to function to tell if the server supports headers.
+func (s *Server) supportsHeaders() bool {
+	if s == nil {
+		return false
+	}
+	s.mu.Lock()
```

Comment:
Either that (getting from s.info) or from options: `return !(s.getOpts().NoHeaderSupport)`

Reply:
That is better, will make that change.

Reply:
Fixed, thanks.